### ISSUE TITLE
mvn clean package Failures (java.lang.RuntimeException: Cannot start Helix Mirror Maker Controller)

### ISSUE THREAD

**Issue by xiagujinqiao** [uber/uReplicator#110]
Hello, when I run mvn clean package, but mvn clean package Failures.

-------------------------------------------------------------------------------

Test set: com.uber.stream.kafka.mirrormaker.controller.rest.ControllerStarterTest

-------------------------------------------------------------------------------
Tests run: 6, Failures: 1, Errors: 0, Skipped: 5, Time elapsed: 4.064 sec <<< FAILURE!
setup(com.uber.stream.kafka.mirrormaker.controller.rest.ControllerStarterTest)  Time elapsed: 3.998 sec  <<< FAILURE!
java.lang.RuntimeException: Cannot start Helix Mirror Maker Controller
        at com.uber.stream.kafka.mirrormaker.controller.rest.ControllerStarterTest.startController(ControllerStarterTest.java:126)
        at com.uber.stream.kafka.mirrormaker.controller.rest.ControllerStarterTest.setup(ControllerStarterTest.java:77)

OS:CentOS Linux release 7.2.1511 (Core)
java:java version "1.8.0_111"
thanks.

**Comment by xhl1988** [comment]
It works fine on my side. Could you please print out the exception you got?

**Comment by xiagujinqiao** [comment]
all log [https://www.dropbox.com/sh/4uaa1y1o0kzzx2k/AADTl4gAChM07yqZOfuBH5jqa?dl=0](url)

**Comment by xhl1988** [comment]
I mean the exception you got in ControllerStarterTest. You can change https://github.com/uber/uReplicator/blob/master/uReplicator-Controller/src/test/java/com/uber/stream/kafka/mirrormaker/controller/rest/ControllerStarterTest.java#L126 to "throw new RuntimeException("Cannot start Helix Mirror Maker Controller", e);" and see what's the exception.

**Comment by xiagujinqiao** [comment]
```
[2018-01-22 16:35:13,578] ERROR Error parsing environment info (com.uber.stream.kafka.mirrormaker.controller.reporter.HelixKafkaMirrorMakerMetricsReporter:55)
[2018-01-22 16:35:13,594] INFO Reporter Metric Prefix is : stats.sjc1.counter.kafka-mirror-maker-controller.sjc1a.localhost (com.uber.stream.kafka.mirrormaker.controller.reporter.HelixKafkaMirrorMakerMetricsReporter:64)
[2018-01-22 16:35:13,663] WARN Trying to connect to Graphite with address: localhost/127.0.0.1:4744 (com.uber.stream.kafka.mirrormaker.controller.reporter.HelixKafkaMirrorMakerMetricsReporter:129)
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.389 sec
Running com.uber.stream.kafka.mirrormaker.controller.TestControllerConf
Configuring TestNG with: TestNG652Configurator
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.255 sec

Results :

Failed tests:
  ControllerStarterTest.setup:77->startController:126 Runtime Cannot start Helix...

Tests run: 23, Failures: 1, Errors: 0, Skipped: 5
[INFO] Finished at: Mon Jan 22 16:35:14 CST 2018
[INFO] Final Memory: 31M/1517M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.14:test (default-test) on project uReplicator-Controller: There are test failures.
[ERROR]
[ERROR] Please refer to /data/uReplicator/uReplicator-Controller/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR]
```

**Comment by xhl1988** [comment]
Could you please just run ControllerStarterTest?

**Comment by xhl1988** [comment]
We have merged #111 to rethrow stacktrace.

**Comment by xiagujinqiao** [comment]
sry,my server environment problems caused,In other environments, compiling is successful.
thanks, close issue.
